libc/include/stddef.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+typedef unsigned long size_t;
```

Comment:
stddef.h is usually provided by the compiler itself and shouldn't be provided here.

Reply:
Removed